doc/release/release_0.17.rst

```diff
@@ -0,0 +1,371 @@
+- Fixing order of elements in docstrings of skimage/color/colorconv (#4518)
+- Fix exposure_adapthist return when clip_limit == 1 (#4519)
+- Adding info on venv activation on Windows (#4521)
+- Fix similarity transform scale (#4524)
+- Added explanation in the example of `segmentation/plot_label.py` to make the background transparent (#4527)
+- Add example code for generating structuring elements. (#4528)
+- blacklist imread version 0.7.2 due to build failure (#4529)
+- Maint: edits to suppress some warnings (unused imports, blank lines) (#4530)
+- MNT: remove duplicate nogil specification (#4546)
+- Blacklist pillow 7.1.0, see #4548 (#4551)
+- Fix binder requirements (#4555)
+- Do not enforce pil plugin in skimage.data (#4560)
+- Remove "backport to 0.14" in github template (#4561)
+- Fix inconsistency in docstring (filters.median) (#4562)
+- Disable key check for texlive in travis-mac as a temporary workaround (#4565)
+- Bump Pywavelets min requirement to 1.1.1 (#4568)
+- Strip backslash in sphinx 3.0.0 (#4569)
+- Remove binary specification from match_descriptors docstring (#4571)
+- Remove importing skimage.transform as tf (#4576)
+- Add note to remove option in doc config when numpydoc will be patched (#4578)
+- update task in TODO.txt (#4579)
+- Rename convert to _convert, as it is a private function (#4590)
+- Do not overwrite data module in plot_skeleton.py (#4591)
+- [CI fix] add import_array in cython files where numpy is cimport-ed (#4592)
+- Recommend cnp.import_array in contribution guide (#4593)
+- Add example of natsort usage in documentation (#4599)
+- Fix broken and permanently moved links (#4600)
+- Fix typo in cython import_array (#4602)
+- Update min required sphinx version for sphinx-copybutton (#4604)
+- Clarify error message when montaging multichannel nD images and multichannel=False (#4607)
+- Fix register_translation warning message (#4609)
+- Add notes on deprecation warnings in marching_cube_* and gray2rgb (#4610)
+- Improve loading speed of our gallery by reducing the thumbnail size (#4613)
+- Fixed wrong behaviour of `exposure.rescale_intensity` for constant input. (#4615)
+- Change math formatting in the docstrings (#4617)
+- Add .mypy_cache to the gitignore (#4620)
+- typo fixes for register rotation gallery example (#4623)
+- Userguide: add a visualization chapter (#4627)
+- Fix deprecation warnings due to invalid escape sequences.  (#4628)
+- add docstring examples for moments_hu and centroid (#4632)
+- Update pooch registry with new file location (#4635)
+- Misleading "ValueError: Input array has to be either 3- or 4-dimensional" in montage (#4638)
+- Fix broken link (#4639)
+- AffineTransform: Allow a single value for 'scale' to apply to both sx & sy (#4642)
+- Fix CI - cython 3.0a4 (#4643)
+- Fix sphinx (#4644)
+- Fix ArrayMap test (#4645)
+- Remove copy of tifffile; install from pip (#4235)
+- Refactor/move neighborhood utility functions in morphology (#4209)
```

Comment:
Man, it's a big release. =D Can we add the PR count to the top of the release summary?